packages/runtime/src/next/index.tsx

```diff
@@ -1,6 +1,7 @@
 export type { PageProps } from './components/page'
 export { Page } from './components/page'
 export { MakeswiftComponent } from './components/MakeswiftComponent'
+export { MakeswiftSlot } from './components/MakeswiftSlot'
```

Comment:
I think we should keep the name as `Slot` instead of `MakeswiftSlot`. It does conflict with the control but they each have different import paths. I'm curious what you think @jamesqquick 

I want us to avoid having every single import have a Makeswift prefix in it

Reply:
I do think separating it somehow from the control would be nice. I guess the difference is where it's imported from? I don't personally mind `MakeswiftSlot` I don't think. What about `SlotComponent`? Is that overly clarifying?

Reply:
Decided on sticking with `Slot` for now.